examples/eqpt_config.json

```diff
@@ -0,0 +1,43 @@
+{     "Edfa":[{
+            "type_variety": "std_medium_gain",
```

Comment:
Can we pick a better name than "type_variety"?

Reply:
Can we also pick a better name than "egpt_config.json"?

Reply:
presented and discussed in previous meetings.